e2e/cluster.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
I think the [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is not necessary because the script here is only used by the source.